### PR TITLE
Standardizing API

### PR DESCRIPTION
**Pull request by wilwade**
- Generate swagger command
- Update controllers to the new standard
- Update graph-sdk
- Remove the api path

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,7 +188,6 @@ Environment files are documented [here](./ENVIRONMENT.md), and a sample environm
 1. Copy the template values into the .env files.
    ```sh
    cp env.template .env
-   cp env.template .env.dev
```
Comment (claireclark1) [inline README.md:191]: why is this gone? assuming we don't have this file anymore?

Comment (wilwade) [reply README.md:191]: Correct. I don't think anything uses `.env.dev` anymore

File: apps/api/src/metadata.ts [hunk 2]
```diff
@@ -1,95 +1,15 @@
...
-              updateGraph: { type: t['../../../libs/common/src/dtos/graph.change.request.reference'].GraphChangeRepsonseDto },
-              watchGraphs: {},
-            },
-          },
-        ],
-      ],
-    },
-  };
-};
+    const t = {
```
Comment (claireclark1) [inline apps/api/src/metadata.ts]: run prettier

**Review by claireclark1 — APPROVED** [review]
🚢

**Pull request merged**